src/python/.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
```

Comment:
ah, answers my previous question, please remove as I don't believe we need this in the shipped version

Reply:
We can keep, gift for future selves..